docs/api.md

```diff
@@ -719,11 +727,56 @@ The default browser context is the only non-incognito browser context.
 
 Creates a new page in the browser context.
 
+
+#### browserContext.overridePermissions(origin, permissions)
+- `origin` <[string]> An [origin] to grant permissions to.
+- `permissions` <[Array]<[string]>> An array of permissions to grant. All permissions that are not listed here will be automatically denied. Permission might be one of the following values:
```

Comment:
Permissions can be one of the following values:

Reply:
Done